packages/components/src/components/carousel/carousel.ts

```diff
@@ -65,6 +65,12 @@ export default class SdCarousel extends SolidElement {
   /** Specifies how many slides should be shown at a given time.  */
   @property({ type: Number, attribute: 'slides-per-page' }) slidesPerPage = 1;
 
+  /**
+   * Specifies the number of slides the carousel will advance when scrolling, useful when specifying a `slides-per-page`
```

Comment:
if changed in the stories, pls also change here.

Reply:
Updated here as well.